hudi-client/hudi-flink-client/src/test/java/org/apache/hudi/testutils/HoodieFlinkClientTestHarness.java

```diff
@@ -58,6 +83,15 @@ public void setTestMethodName(TestInfo testInfo) {
     }
   }
 
+  @org.junit.jupiter.api.BeforeEach
```

Comment:
let's remove this, we don't have to call this every time.

Reply:
sorry if I mislead you on this comment. I was trying to say we should remove this `BeforeEach` here since we don't have to run this `setUp` before all the test cases.